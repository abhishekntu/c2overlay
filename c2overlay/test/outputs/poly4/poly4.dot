digraph graphname {
N8 [color="black", label="N6N9_add_N6_mul_N9_N8"];
N9 [color="black", label="O0_N12_N9"];
N1 [color="black", label="I0_N1_N1"];
N2 [color="black", label="I1_N2_N2"];
N3 [color="black", label="I2_N3_N3"];
N4 [color="black", label="I3_N4_N4"];
N5 [color="black", label="I4_N5_N5"];
N6 [color="black", label="N10N7N11_add_N10_mul_N7_add_N11_N6"];
N7 [color="black", label="N8_mul_N8_N7"];
N8 -> N6;
N1 -> N7;
N2 -> N8;
N2 -> N7;
N3 -> N6;
N4 -> N8;
N5 -> N6;
N6 -> N9;
N7 -> N6;
}
