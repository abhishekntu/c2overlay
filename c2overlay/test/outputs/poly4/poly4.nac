procedure foo(in s32 a,in s32 c,in s32 h,in s32 m,in s32 n,out s32 D_2449)
{
  localvar s32 t1_1;
  localvar s32 t1_2;
  localvar s32 t1_3;
  localvar s32 t1_4;
  localvar s32 t1_5;
  localvar s32 t1_6;
  localvar s32 t1_7;
  localvar s32 t1_8;
  localvar s32 t1_9;
  localvar s32 t1_10;
  localvar s32 t2_1;
  localvar s32 t2_2;
  localvar s32 t2_3;
  localvar s32 t2_4;
  localvar s32 t2_5;
  localvar s32 t2_6;
  localvar s32 t2_7;
  localvar s32 t2_8;
  localvar s32 t3_1;
  localvar s32 t3_2;
  localvar s32 t3_3;
  localvar s32 t3_4;
  localvar s32 t3_5;
  localvar s32 t3_6;
  localvar s32 t3_7;
  localvar s32 t4_1;
  localvar s32 t4_2;
  localvar s32 t4_3;
  localvar s32 t4_4;
  localvar s32 t5_1;
  localvar s32 t5_2;
  localvar s32 t5_3;
  localvar s32 t6_1;
  localvar s32 t6_2;
  localvar s32 t6_3;
  localvar s32 t7_1;
  localvar s32 t7_2;
  localvar s32 t8_1;
  localvar s32 t8_2;
  localvar s32 t9_1;
  localvar s32 t10_1;
  localvar s32 t11_1;
L0005:
  t1_1 <= mul a,c;
  t2_1 <= add m,t1_1;
  t3_1 <= add c,n;
  t3_2 <= mul t2_1,h;
  t4_1 <= mul t3_1,c;
  D_2449 <= add t4_1,t3_2;
}
