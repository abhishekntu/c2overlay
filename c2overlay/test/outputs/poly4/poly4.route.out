Array size: 5 x 5 logic blocks.

Routing:

Net 0 (N1)

SOURCE (2,0)  Pad: 1  
  OPIN (2,0)  Pad: 1  
 CHANX (2,0)  Track: 1  
  IPIN (2,1)  Pin: 2  
  SINK (2,1)  Class: 0  


Net 1 (N2)

SOURCE (3,0)  Pad: 1  
  OPIN (3,0)  Pad: 1  
 CHANX (3,0)  Track: 1  
 CHANY (2,1)  Track: 0  
  IPIN (2,1)  Pin: 1  
  SINK (2,1)  Class: 0  
 CHANY (2,1)  Track: 0  
 CHANX (2,1)  Track: 1  
 CHANX (1,1)  Track: 1  
  IPIN (1,1)  Pin: 0  
  SINK (1,1)  Class: 0  


Net 2 (N3)

SOURCE (0,3)  Pad: 1  
  OPIN (0,3)  Pad: 1  
 CHANY (0,3)  Track: 0  
 CHANX (1,3)  Track: 0  
 CHANY (1,3)  Track: 1  
 CHANX (1,2)  Track: 1  
  IPIN (1,2)  Pin: 0  
  SINK (1,2)  Class: 0  


Net 3 (N4)

SOURCE (1,0)  Pad: 1  
  OPIN (1,0)  Pad: 1  
 CHANX (1,0)  Track: 1  
  IPIN (1,1)  Pin: 2  
  SINK (1,1)  Class: 0  


Net 4 (N5)

SOURCE (0,2)  Pad: 1  
  OPIN (0,2)  Pad: 1  
 CHANY (0,2)  Track: 0  
  IPIN (1,2)  Pin: 3  
  SINK (1,2)  Class: 0  


Net 5 (N8)

SOURCE (1,1)  Class: 1  
  OPIN (1,1)  Pin: 4  
 CHANX (1,1)  Track: 0  
  IPIN (1,2)  Pin: 2  
  SINK (1,2)  Class: 0  


Net 6 (N9)

SOURCE (1,2)  Class: 1  
  OPIN (1,2)  Pin: 7  
 CHANY (0,2)  Track: 1  
 CHANY (0,1)  Track: 1  
  IPIN (0,1)  Pad: 0  
  SINK (0,1)  Pad: 0  


Net 7 (N7)

SOURCE (2,1)  Class: 1  
  OPIN (2,1)  Pin: 7  
 CHANY (1,1)  Track: 0  
 CHANY (1,2)  Track: 0  
  IPIN (1,2)  Pin: 1  
  SINK (1,2)  Class: 0  
